testrunner/src/mill/testrunner/TestRunner.scala

```diff
@@ -140,28 +141,6 @@ object TestRunner {
       )
       s"@${argsFile.toString()}"
     }
-    private def copy(
-        framework: String = framework,
-        classpath: Seq[String] = classpath,
-        arguments: Seq[String] = arguments,
-        sysProps: Map[String, String] = sysProps,
-        outputPath: String = outputPath,
-        colored: Boolean = colored,
-        testCp: String = testCp,
-        homeStr: String = homeStr,
-        globSelectors: Seq[String] = globSelectors
-    ): TestArgs = new TestArgs(
-      framework,
-      classpath,
-      arguments,
-      sysProps,
-      outputPath,
-      colored,
-      testCp,
-      homeStr,
-      globSelectors
-    )
-
```

Comment:
We made this one private and some other tricks to keep the option to evolve this class in a binary compatible way. What's the intention of changing that?

Reply:
This is still WIP, but I want to try to adjust the code path so we don't need to use a `case class` here. That would make maintaining binary compatibility much simpler